src/classes/groundplaces.ts

```diff
@@ -27,23 +40,34 @@ export class GroundPlaces {
   /**
    * @description Create the stopGroup with the values given.
    * @param {StopClusterGpuid} stopClusterGpuid - Ground place unique identifier of the stopCluster parent.
-   * @param {StopGroup} stopGroupInfos - StopGroup informations.
-   * @returns {StopGroup}
+   * @param {StopInfos} stopGroupInfos - StopGroup informations.
+   * @returns {void|Error}
    */
   // @ts-ignore
-  public createStopGroup(stopClusterGpuid: StopClusterGpuid, stopGroupInfos: StopGroup): void {}
+  public createStopGroup(stopClusterGpuid: StopClusterGpuid, stopGroupInfos: StopInfos): void | Error {
+    // Generate StopGroup with ground place unique identifier generator
```

Comment:
i dont think these comments are really useful, and function names are really clear so personally i would remove them. lets see what others think

Reply:
👍

Reply:
Removed